src/utils/checkFileType.ts

```diff
@@ -0,0 +1,15 @@
+import { FileExtension, fileMimeType } from '@/constants/file';
+
+/**
+ * 주어진 FileList에서 모든 파일이 types 확장자 목록에 해당하는지 검사하는 함수
+ *
+ * @param files 검사할 `FileList` 객체
+ * @param types 허용할 파일 확장자 목록 (예: ['.pdf', '.png'])
+ * @returns 모든 파일이 허용된 확장자에 해당하면 `true`, 그렇지 않으면 `false`
+ *
+ */
+export const checkFileType = (files: FileList, types: FileExtension[]) => {
```

Comment:
(취향 차이)
```suggestion
export const validateFileType = (files: FileList, types: FileExtension[]) => {
```
check도 충분히 좋네요~ **검사** 에 비중을 더 두면 check, **허용된**에 비중을 더 두면 validate 정도로 생각하시면 될 거 같아요~

Reply:
return 값이 boolean이고 허용에 좀 더 가까워서 validate가 더 좋을 것 같아요! 수정해둘게요~!
제안 감사합니다!